spec.md

```diff
@@ -143,9 +143,40 @@ A web-friendly display name is camel-cased with spaces and punctuation supported
 | id*  | string  | An identifier used to correlate this plan in future requests to the broker. This must be globally unique within a platform marketplace. Using a GUID is recommended.  |
 | name*  | string  | The CLI-friendly name of the plan. Must be unique within the service. All lowercase, no spaces.  |
 | description*  | string  | A short description of the plan.  |
-|  metadata | JSON object  | An opaque object of metadata for a service plan. Controller treats this as a blob. Note that there are (conventions)[https://docs.cloudfoundry.org/services/catalog-metadata.html] in existing brokers and controllers for fields that aid in the display of catalog data. |
+|  metadata | object  | A list of metadata for a service plan.  |
 | free  | boolean  | When false, instances of this plan have a cost. The default is true  |
 |  bindable | boolean  |  Specifies whether instances of the service plan can be bound to applications. This field is optional. If specified, this takes precedence over the <tt>bindable</tt> attribute of the service. If not specified, the default is derived from the service. |
+| [schemas](#SObject) | object | Schema definitions for service instances and bindings for the plan. |
+
+
+##### Schema Object <a name="SObject"></a> #####
```

Comment:
Nope; I was just being consistent with the current spec (I know there is an outstanding PR that addresses this). I'll fix in this branch!